odh-dashboard/base/kustomization.yaml

```diff
@@ -15,3 +15,7 @@ resources:
 - routes.yaml
 - service.yaml
 - rhodsquickstartcrd.yaml
+images:
+- name: rhods-dashboard
+  newName: quay.io/opendatahub/rhods-dashboard
+  newTag: latest
```

Comment:
@LaVLaS shouldn't we avoid using the `latest` tag here? Or does this get replaced elsewhere?

Reply:
I left this in for consistency but this is never parsed by kfctl/kustomize because we have a "hack" in place during the build process that runs `sed` on `spec.contaners[].image:` for every `Deployment / DeploymentConfig / StatefulSet / ...`  to replace the image url with the CPaaS built image url

Reply:
Got it. Thanks!